### DISCUSSION TITLE
How to change town setup

### DISCUSSION THREAD

**Discussion by falconavia** [TownyAdvanced/Towny#5931] [Q&A] [answered]
Hi there!

I'm in a small Towny server, and recently a lot of people have been complaining about how little claim plots there are. The owner said that he enabled "cities and metropolises" and had metropolises on 2,5k claims. I was wondering how to change my town type, as right now I only have 26 claims, and my Google search came up short. Any help would be nice! Thanks.

**Comment by LlmDl** [comment, marked as answer]
Get more residents in your town to get a higher level of town. Higher levels of town will get you more potential claims.
